README.md

```diff
@@ -69,16 +76,11 @@ run pingbacks, add `PINGBACKS=true` to your `.env` file.
 
 # Deployment
 
-This repo actually represents 2 lambda functions: `analytics-ingest-ENVIRONMENT`
-to insert downloads/impressions into bigquery, and `analytics-pingback-ENVIRONMENT`
-to manage Adzerk impressions, other HTTP pingbacks, and realtime redis increments.
+The 3 lambdas functions are deployed via a Cloudformation stack in the [Infrastructure repo](https://github.com/PRX/Infrastructure/blob/master/stacks/analytics-ingest-lambda.yml):
 
-These need separate functions due to the retry logic of kinesis-triggered lambdas.
-If any errors are thrown, that same kinesis segment will be retried over and over,
-until they succeed or expire. For BigQuery, this is fine, since the `insertId`
-does some de-duping for us. But for pingbacks, we want to just allow these to
-fail without re-running the entire kinesis segment. This prevents one misbehaving
-pingback from causing a bunch of duplicates GETs on the rest of them.
+ - `AnalyticsBigqueryLambda` - insert downloads/impressions/bytes into BigQuery
+ - `AnalyticsPingbacksLambda` - ping Adzerk impressions and 3rd-party pingbacks
+ - `AnalyticsRedisLambda` - realtime Redis increments
```

Comment:
Curious, Is bundling these together just a repo/deployment convenience?

Reply:
Yeah, to share some of the overhead of reading/decoding kinesis, logging, error-handling, etc.

Also, I think I started this repo thinking there would just be one function.  But then I ran into (1) can't access Redis unless the lambda is confided to the VPC, and (2) kinesis batch retry logic for idempotent bigquery is different than http pingbacks.  So ended up having to split it out.